corehq/apps/users/templates/users/web_users.html

```diff
@@ -17,14 +18,21 @@
       However, these users cannot submit forms from phones
       using the {{ cc_name }} Application.
     {% endblocktrans %}
-    {% if not request.is_view_only %}
+
       <div class="btn-toolbar">
+        {% if not request.is_view_only %}
         <a class="btn btn-primary"
            href="{% url "invite_web_user" domain %}">
           <i class="fa fa-plus"></i> {% trans 'Invite Web User' %}
         </a>
+        {% endif %}
+
+        {% if can_access_all_locations %}
```

Comment:
I didn't know location-restricted users had access to `ListWebUsersView`. Do you know how that works? Are the visible web users filtered by location?

Reply:
Looked at the [docs](https://confluence.dimagi.com/display/commcarepublic/Location-Based+Data+Access+and+User+Editing+Restrictions), and you're right that location-restricted users do not have access to `ListWebUsersView` and this is only applicable to mobile workers.

Reply:
That's correct, they don't have access to that web users.  I wouldn't trust the docs to be comprehensive for that sort of thing though - you can look at the view function and see whether it's decorated with `@location_safe` (which it isn't).  You can read a bit about how that works [here](https://commcare-hq.readthedocs.io/locations.html#location-permissions) (pulled from `locations/permissions.py`)

Reply:
@esoergel thank you that clarifies a lot of things! I'll get rid of the unnecessary checks